src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -588,6 +607,18 @@ public function run() {
 
         echo pht('Updated an existing Differential revision:')."\n";
       } else {
+        // UBER CODE
+        if ($wip && !idx($revision['fields'], 'plan-changes', false)) {
+          $prompt = 'Keyword "wip" was detected, do you want to set '.
```

Comment:
Same here:
`Keyword "wip" found on revision title. Do you want to ...`

Also "or by changing title...' (remove "change").